Dockerfile

```diff
@@ -2,8 +2,13 @@
 # Copyright 2021-2021 VMware, Inc.
 # SPDX-License-Identifier: Apache-2.0
 #
-FROM bitnami/node:12-prod
-LABEL maintainer "Bitnami <containers@bitnami.com>"
+FROM bitnami/node:16-prod
```

Comment:
AFAIK we don't maintain `node:16-prod` anymore, only `node:16`: https://github.com/bitnami/containers/tree/main/bitnami/node/